bigbluebutton-html5/imports/ui/components/chat/service.js

```diff
@@ -193,6 +195,13 @@ const sendMessage = (receiverID, message) => {
     from_color: 0,
   };
 
+  let currentClosedChats = Storage.getItem('closedChatList');
```

Comment:
You probably want to store the item key on a constant, since you are using on multiple parts of the code.

```
const CLOSED_CHAT_LIST_KEY = 'closedChats';
let closedChats = Storage.getItem(CLOSED_CHAT_LIST_KEY);
```